src/sass/mixins/private/_button-mixins.scss

```diff
@@ -101,3 +101,12 @@ $button-border-radius-large: 24px;
         @include btn-fixed-height();
     }
 }
+
+@mixin btn-animations() {
+    transition: all var(--motion-duration-short-3)
```

Comment:
This is less performant than being specific about what we're transitioning. We should use `transform`.

Reply:
We need to transition on 
Color
Background color
Filters 
And transform. 
To me this is much cleaner then typing all those out.